### PR TITLE
Dennis zack branch

### PR DESCRIPTION
**Pull request by DennisZack**
Fully completed code. Let me know if you need any help merging or understanding what I did.

### CODE REVIEW

File: app/notifications.py [hunk 1]
```diff
@@ -1,23 +1,71 @@
...
+        'able4110group@gmail.com',
+        'able4110group@gmail.com',
+        "Test Message, Do Not Respond")  # Accesses the server function to send the email
+      server.quit()  # Exits out of the server so there is no trailing connection
+      print('Email sent!')  # Let the user know that their email was sent sussessfully
+
+    except:
+      print('Something went wrong...')
+
+def main(self):
```
Comment (KenGajefski) [inline app/notifications.py:66]: What is this?

File: app/notifications.py [hunk 2]
```diff
@@ -1,23 +1,71 @@
 import smtplib
+import abc
 
+class Invoker:
```
Comment (KenGajefski) [inline app/notifications.py:4]: Why is this here if it's empty?

File: app/routes.py [hunk 3]
```diff
@@ -2,9 +2,9 @@
 import sqlite3
 from app import app, db
 from flask import render_template, redirect, url_for
-from app.notifications import Email
 from app.models import User, Reviews
-from app.forms import AccountCreation, ReviewCreation
+from app.forms import AccountCreation, ReviewCreation\
```
Comment (KenGajefski) [inline app/routes.py]: Remove the '\' character.

File: app/routes.py [hunk 4]
```diff
@@ -2,9 +2,9 @@
...
+    #, ReviewDisplay
```
Comment (KenGajefski) [inline app/routes.py]: Why is this on a second line and commented out? Delete if not needed.

File: app/routes.py [hunk 5]
```diff
@@ -21,19 +21,18 @@ def write_review():
         review = Reviews(review=new_review.review.data, location=1, user='gh')
         db.session.add(review)
         db.session.commit()
-        return redirect('/')
+        #return redirect('/')
```
Comment (KenGajefski) [inline app/routes.py]: Why is this commented out?

File: app/routes.py [hunk 6]
```diff
@@ -55,21 +54,21 @@ def registration():
...
-    send.send_email()
-    return render_template('main.html')
-
-@app.route('/see_editor_picks', methods=['GET', 'POST'])
-def retrieve_editor_picks():
-    return render_template('editor-picks.html')
+@app.route('/user_profile', methods=['GET', 'POST'])
+def retrieve_user_profile():
+    #review_display = ReviewDisplay()
+    #review_display.display_database_reviews()
```
Comment (KenGajefski) [inline app/routes.py]: If these are not needed, delete them.

File: app/routes.py [hunk 7]
```diff
@@ -55,21 +54,21 @@ def registration():
...
+    return render_template('User Profile.html')
 
-@app.route('/return_to_main', methods=['GET', 'POST'])
+@app.route('/return_to_menu', methods=['GET', 'POST'])
 def return_to_main_menu():
     return render_template('main.html')
```
Comment (KenGajefski) [inline app/routes.py:62]: Not necessary. Look at using an if statement and the redirect method from Flask.

File: app/routes.py [hunk 8]
```diff
@@ -55,21 +54,21 @@ def registration():
...
 
+@app.route('/show_reviews', methods=['GET', 'POST'])
+def list_all_reviews():
+    all_reviews = Reviews.query
```
Comment (KenGajefski) [inline app/routes.py:66]: Is this a valid query statement? Does this end up passing anything?

File: app/templates/Owner Profile.html [hunk 9]
```diff
@@ -0,0 +1,21 @@
...
+    <p>Temp</p><br><br><br>
+    <button name="test" type="button">Click Me!</button>
+    <br><br><br>
+
+    <form action="return_to_main">
+        <button type="submit">Return to Menu</button>
+    </form>
+
+</body>
+</html>
```
Comment (KenGajefski) [inline app/templates/Owner Profile.html]: Needs to extend base.html

File: app/templates/Owner Profile.html [hunk 10]
```diff
@@ -0,0 +1,21 @@
...
+<head>
+    <meta charset="UTF-8">
+    <title>Business Owner Profile</title>
+</head>
+<body>
+
+    <h1>Business Owner Profile</h1><br>
+
+    <p>Temp</p><br><br><br>
+    <button name="test" type="button">Click Me!</button>
```
Comment (KenGajefski) [inline app/templates/Owner Profile.html:14]: Get rid of this button if it has no functionality.

**Review by KenGajefski — COMMENTED** [review]
Pull from master first then do all suggested changes.

**Comment by KenGajefski** [comment]
@DennisZack Please merge in master and fix all comments I've left so I can merge this branch before it is finally due.

**Comment by DennisZack** [comment]
I'll have all your comments resolved by Monday night.
